### ISSUE TITLE
.Gulden in home directory is capitalized

### ISSUE THREAD

**Issue by BioMike** [Gulden/gulden-dev#50]
The new ".Gulden" folder in the home directory is capitalized, which is an annoyance on systems which use case-sensitive file systems. I would like to suggest to change it to ".gulden", to make it consistent in use like other programs.

**Comment by mjmacleod** [comment]
Hi,

We did discuss this internally before making the change.

While I am aware that some feel this way and there is somewhat of 'convention' towards this on some distros, at the same time it is largely a matter of opinion what is better here.
If I look on the average Linux distro I can find many examples of programs (especially cross platform ones) that use uppercase and if I do a web search, for every person saying they don't like uppercase I can find someone saying they prefer it.

In terms of consistency there are many ways to be consistent, (you are of course talking about being consistent with platform conventions) in this case we chose the option of being consistent (1) Internally with our own program (2) Across all platforms
For this Uppercase everywhere was decided to be the best, there are of course trade offs but we feel that this type of consistency is more important for our overall user experience than any Linux specific 'conventions' that are in place here.
Fortunately for those who feel strongly about this there is the option to pass the -datadir command line flag with which they can set the directory to whatever they want.

I will leave this bug here in case others want to give their opinions, or if we want to change our mind in future but this is where we want things to be for now.

**Comment by BioMike** [comment]
Hi,

Fair enough, but I just want to add this here (for others, because a discussion regarding this seems not possible any more):

The "." in the data directory on Linux is not present for both Windows or Mac (util.cpp), so even internally within the program, across all platforms, it is not consistent to begin with. Of the +40 programs (many cross -platform) in my home folder that have a data directory, only Gulden uses a capitalized data directory.

**Comment by mjmacleod** [comment]
To name a few common examples of programs not doing this, by no means exhaustive:
BOINC, Skype, ABISuite, VirtualBox... And then you have many more that are capatalised but have already moved to the .local directory - e.g. Keepass.

Whats of much more interest to me is that if we look at e.g. the .config directory (as an example) the average system has more things capitalized here than not - e.g. "DIgiByte" is capitalized here but their home directory is not, and this is the sort of consistency that is far more important to the dev team.

A much more interesting topic (in my opinion) is whether the folder should move from ~/.Gulden to ~/.local/Gulden

**Comment by BioMike** [comment]
I agree that the .config contains things which are capitalized. But isn't .config intended for graphical programs (isn't it part of the opendesktop specs)? How to handle the non-graphical daemon? But then again guldend (?) should be using /var/lib as system wide daemon any way.
